addons/mail/static/src/core/common/suggestion_hook.js

```diff
@@ -110,9 +109,7 @@ export function useSuggestion() {
                 });
             }
             if (option.cannedResponse) {
-                comp.props.composer.cannedResponses.push(
-                    store.CannedResponse.insert(option.cannedResponse)
-                );
+                comp.props.composer.cannedResponses.push(option.cannedResponse);
```

Comment:
Any reason to use push rather than add?
I feel like add is almost always better?

Reply:
Except if posting a message containing a canned response more than once should pass the id as many times as there are occurrences. `push()` allow to put same content more than once, while `add()` does not add if it's already in the relation